go/interpreter/lfvm/instructions_test.go

```diff
@@ -208,43 +208,85 @@ func TestPush4(t *testing.T) {
 }
 
 func TestCallChecksBalances(t *testing.T) {
```

Comment:
I am having a hard time figuring out what this test does. Please improve the name of the test, also for other cases.